src/libs/actions/PolicyConnections.ts

```diff
@@ -0,0 +1,38 @@
+import Onyx from 'react-native-onyx';
+import type {OnyxUpdate} from 'react-native-onyx';
+import * as API from '@libs/API';
+import type {OpenPolicyAccountingPageParams} from '@libs/API/parameters';
+import {READ_COMMANDS} from '@libs/API/types';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+function openPolicyAccountingPage(policyID: string) {
+    const hasConnectionsDataBeenFetchedKey = `${ONYXKEYS.COLLECTION.POLICY_HAS_CONNECTIONS_DATA_BEEN_FETCHED}${policyID}` as const;
```

Comment:
Why is `as const` necessary here?

Reply:
fix: https://github.com/Expensify/App/pull/39132/commits/a084e5295dea50b4d14fb560c2ad5e58e83cb1f0

Reply:
It looks like you removed it from a different spot. Is it still necessary here?

Reply:
The `const` is necessary here, otherwise the `hasConnectionsDataBeenFetchedKey` gets `string` as a type and then `OnyxUpdate` is not happy with such key:

<img width="1009" alt="image" src="https://github.com/Expensify/App/assets/87341702/5c897a77-a02f-479c-8d99-e15d2729858c">

Reply:
haha I definitely moved the `const` from a different place but it didn't cause the TS error 😆 for this particular one